tests.py

```diff
@@ -249,6 +249,25 @@ def doctest_short_repr_unbound_method():
 
     """
 
```

Comment:
Nitpick: [PEP-8](https://www.python.org/dev/peps/pep-0008/) specifies two blank lines between top-level functions.

I didn't catch this earlier so instead of making you jump through hoops now I think I'll just merge it and fix it later.